source/lib/rocprof-sys/library/amd_smi.cpp

```diff
@@ -143,6 +143,7 @@ data::sample(uint32_t _dev_id)
     auto _ts = tim::get_clock_real_now<size_t, std::nano>();
     assert(_ts < std::numeric_limits<int64_t>::max());
     amdsmi_gpu_metrics_t _gpu_metrics;
+    bool                 is_vcn_or_jpeg_activity_enabled = false;
```

Comment:
```suggestion
    bool                 _vcn_or_jpeg_activity_enabled = false;
```
Small change. Just renamed to be consistent with the format of other variables declared in the function